drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/DecisionTableXLSToDecisionTableGuidedConverterTest.java

```diff
@@ -245,7 +244,7 @@ public void testImportGeneration() {
         verify(importsService,
                times(1)).save(projectImportsPathArgumentCaptor.capture(),
                               projectImportsArgumentCaptor.capture(),
-                              any(Metadata.class),
```

Comment:
@caponetto I suggest here to use `isA(Metadata.class)`, in order don't lost the already present bound on  `Metadata` class as third parameter of `save` method verification.